conjure-undertow-lib/src/main/java/com/palantir/conjure/java/undertow/lib/UndertowService.java

```diff
@@ -16,13 +16,19 @@
 
 package com.palantir.conjure.java.undertow.lib;
 
+import java.util.List;
+
 /**
- * Creates a {@link Registrable} which may be registered with a web server. The server is responsible
- * for providing a {@link ServiceContext} allowing API implementors to add APIs using
+ * A {@link UndertowService} provides a list of {@link Endpoint endpoints} when it is
+ * given the {@link UndertowRuntime}. These {@link Endpoint endpoints} may be
+ * registered with an {@link EndpointRegistry} in order to be exposed by a
+ * web server. The server is responsible for providing an {@link UndertowRuntime} and
+ * orchestrating registration with the {@link EndpointRegistry} allowing API
+ * implementors to add APIs using
  * <code>server.api(MyServiceEndpoints.of(myServiceImpl)</code>.
  */
-public interface Service {
+public interface UndertowService {
 
-    Registrable create(ServiceContext context);
+    List<Endpoint> endpoints(UndertowRuntime runtime);
```

Comment:
should this be `createEndpoints`, or are we happy with the `endpoints` method name?

Reply:
I think 'endpoints' is probably fine?